src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/TestRulesContext.cs

```diff
@@ -72,9 +73,39 @@ public static TestRulesContext CreateAsync(Network network, [CallerMemberName]st
             testRulesContext.Chain = new ConcurrentChain(network);
 
             NodeDeployments deployments = new NodeDeployments(testRulesContext.Network, testRulesContext.Chain);
-            testRulesContext.Consensus = new ConsensusRules(testRulesContext.Network, testRulesContext.LoggerFactory, testRulesContext.DateTimeProvider, testRulesContext.Chain, deployments, consensusSettings, testRulesContext.Checkpoints).Register(new FullNodeBuilderConsensusExtension.CoreConsensusRules());
+            testRulesContext.Consensus = new PowConsensusRules(testRulesContext.Network, testRulesContext.LoggerFactory, testRulesContext.DateTimeProvider, testRulesContext.Chain, deployments, consensusSettings, testRulesContext.Checkpoints).Register(new FullNodeBuilderConsensusExtension.PowConsensusRulesRegistration());
 
             return testRulesContext;
```

Comment:
Since all the methods are static maybe make TestRulesContextFactory a static class so it can't be instantiated?

Reply:
ah yeah sure